contracts/Origins/OriginsBase.sol

```diff
@@ -1075,10 +1117,19 @@ contract OriginsBase is IOrigins, OriginsEvents {
 	 * @return true if the stake is enough, false otherwise.
 	 */
 	function checkStakesByTier(uint256 _tierID, address _userAddress) external view returns (bool) {
-		if (_userAddress == address(0)) {
-			_checkStakesByTier(_tierID, msg.sender);
-		} else {
-			_checkStakesByTier(_tierID, _userAddress);
-		}
+		address _userAddr = _userAddress == address(0) ? msg.sender : _userAddress;
+		_checkStakesByTier(_tierID, _userAddr);
+	}
```

Comment:
this function always returns false (missed `return` statement)

Reply:
Solved in 43c2bdd